aws-ecsfargate-terraform/README.md

```diff
@@ -1,72 +1,117 @@
-# Deploying the 1Password SCIM Bridge in AWS ECS with Terraform
+# Deploying the 1Password SCIM Bridge on AWS Fargate with Terraform
 
-This guide will run you through a deployment of the 1Password SCIM bridge to your Amazon Web Service Elastic Container Service (ECS) Fargate using Terraform. 
+This guide will run you through a deployment of the 1Password SCIM bridge on AWS Fargate using Terraform. 
 
-Note that due to the highly advanced and customizable nature of Fargate, this is only a suggested starting point. You may modify it to your needs to fit within your existing infrastructure.
+Note that due to the highly advanced and customizable nature of Amazon Web Services, this is only a suggested starting point. You may modify it to your needs to fit within your existing infrastructure.
 
 ## Prerequisites
 
 Before beginning, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there.
 
-- the [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) command line tools
+- [Terraform](https://www.terraform.io/downloads)
 - `scimsession` file and bearer token (as seen in `PREPARATION.md`)
-- (Optional) DNS Zone in Route53
 
-## Log In With `aws`
+## Sign in with `aws`
 
 Ensure you are authenticated with the `aws` tool in your local environment.
 
 See [Terraform AWS Authentication](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication) for more details.
 
 ## Configuration
 
-### Copy Configuration
+### Copy configuration template
 
 Copy `terraform.tfvars.template` to `terraform.tfvars`:
 
 ```bash
 cp terraform.tfvars.template terraform.tfvars
 ```
 
-### Copy `scimsession` File
+### Copy `scimsession` file
 
-Copy the `scimsession` file in the terraform code directory:
+Copy the `scimsession` file in the Terraform code directory:
 
 ```bash
-cp /path/to/scimsession ./
+cp <path>/scimsession ./
 ```
 
-This will install the `scimsession` file automatically in your instance.
+This will automatically create an AWS secret containing the contents of the `scimsession` file in your instance.
 
-NOTE: If you skip this step or the installation of the `scimsession` file is not successful, you can perform this step manually afterwards. Ensure you `base64` encode the `scimsession` file, and store it in a secret as plain text (not in json, and not wrapped in quotation marks).
+**Note:** If you skip this step or the installation of the scimsession file is not successful, you can create the required AWS secret manually. Ensure you `base64` encode the `scimsession` file, and store it in a secret as **plain text** (not as JSON, and not wrapped in quotation marks):
 
 ```bash
 # only required if the automatic installation of the 'scimession' file is not successful
-cat /path/to/scimsession | base64
+cat <path>/scimsession | base64
```

Comment:
I'm wondering if this is more clear than the previous version. Can I ask why we changed it?

Reply:
I'm trying to signify a placeholder, highlighting exactly what characters need to be replaced, so people don't blindly copy/paste, or replace any more or less than they need to. Angle brackets are pretty intuitive to me for this, but I will happily defer to our best practices for this. :-)

Reply:
Cool! Happy with it as is. As long as it doesn't work when you copy and paste and forget to update it, I'm happy. 🙂